### PR TITLE
Organize leak tracking TODOs.

### PR DESCRIPTION
**Pull request by polina-c**
Prerequisite: https://github.com/flutter/flutter/issues/141119m should merge to G3 (done).

This PR:
1. Adds/removes opt outs for leak tracking
2. Adds tag `[leaks-to-clean]` to all remaining opt outs that should be fixed, to make the search easier

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -5096,7 +5096,7 @@ targets:
 
   - name: Windows framework_tests_libraries_leak_tracking
     recipe: flutter/flutter_drone
-    bringup: true # New target: https://github.com/flutter/flutter/issues/140414
+    # bringup: true # New target: https://github.com/flutter/flutter/issues/140414
```
Comment (goderbauer) [inline .ci.yaml:5099]: Is this change supposed to be in here?

File: packages/flutter/test/animation/live_binding_test.dart [hunk 2]
```diff
@@ -79,7 +79,7 @@ void main() {
   }, skip: true); // Typically skip: isBrowser https://github.com/flutter/flutter/issues/42767
 
   testWidgets('Should show event indicator for pointer events with setSurfaceSize',
-  // TODO(polina-c): clean up leaks, https://github.com/flutter/flutter/issues/134787
+  // TODO(polina-c): clean up leaks, https://github.com/flutter/flutter/issues/134787 [leaks-to-clean]
```
Comment (goderbauer) [inline packages/flutter/test/animation/live_binding_test.dart:82]: Looks like they all have the same issue attached, could you just search for that?

Comment (polina-c) [reply packages/flutter/test/animation/live_binding_test.dart:82]: Nope, they are different. There are at least two. And, when we start investigating, new ones will show up.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**